app/src/main/res/layout/topic_practice_footer_view.xml

```diff
@@ -23,15 +23,16 @@
       android:layout_width="160dp"
       android:layout_height="wrap_content"
       android:layout_marginTop="36dp"
-      android:background="@{isSubmitButtonActive ? @drawable/state_button_primary_background :@drawable/start_button_transparent_background}"
+      android:background="@{isSubmitButtonActive ? @drawable/state_button_primary_background :@drawable/state_button_inactive_background}"
       android:clickable="@{isSubmitButtonActive}"
       android:fontFamily="sans-serif-medium"
       android:minHeight="36dp"
       android:text="@string/topic_practice_start"
-      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonInactive}"
+      android:layout_marginBottom="108dp"
+      android:textColor="@{isSubmitButtonActive ? @color/white : @color/submitButtonTextInactive}"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done